pkg/steps/source.go

```diff
@@ -53,6 +53,7 @@ RUN git submodule update --init
 
 type sourceStep struct {
 	config             api.SourceStepConfiguration
+	gitConfig          *api.InputGitSourceStepConfiguration
```

Comment:
The `sourceStep` is what creates the `src` tag in the `pipeline` imagestream. The test base image is _not_ the same.